File: Python/URI/1134.py
cliente = 0
alcool = 0
gasolina = 0
diesel = 0

while (cliente != 4):

    cliente = int(input())

    if (cliente == 2):
        gasolina = gasolina + 1
    elif (cliente == 1):
        alcool = alcool + 1

    elif (cliente == 3):
        diesel = diesel + 1

    elif (cliente == 4):
        break

print("MUITO OBRIGADO")
print("Alcool: " + str(alcool))
print("Gasolina: " + str(gasolina))
print("Diesel: " + str(diesel))

File: JS/URI/1134.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var cliente;
var alcool = 0;
var gasolina = 0; 
var diesel = 0;
         
while (cliente != 4) {

    cliente = parseInt(lines.shift());
    switch (cliente) {
        case 1:
            alcool++;
            break;
        case 2:
            gasolina++;
            break;
        case 3:
            diesel++;
            break;
        case 4:
            cliente = 4;
            break;
    }
}

console.log("MUITO OBRIGADO");
console.log("Alcool: " + alcool);
console.log("Gasolina: " + gasolina);
console.log("Diesel: " + diesel);